{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2018-10870",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://www.securityfocus.com/bid/104857",
          "name": "104857",
          "refsource": "BID",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:2373",
          "name": "RHSA-2018:2373",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10870",
          "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10870",
          "refsource": "CONFIRM",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "redhat-certification does not properly sanitize paths in rhcertStore.py:__saveResultsFile. A remote attacker could use this flaw to overwrite any file, potentially gaining remote code execution."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-07-19T22:29Z",
  "lastModifiedDate": "2018-08-11T01:29Z"
}